addons/website/views/snippets/snippets.xml

```diff
@@ -742,15 +742,19 @@
         </we-select>
 
         <!-- Generic Hamburger Options (mobile or not) -->
-        <we-select string="Hamburger Type"
+        <we-select string="Mobile Menu"
```

Comment:
Re-check with SBU, this label seems just wrong for hamburger templates, it was debated a lot last year :grimacing:

Reply:
@sbuhl  is requested at reception.

https://www.awesomescreenshot.com/image/7275087?key=a4608b5ae3183402a37c2f509040b6dc

Reply:
8 days ago and you are asking for a review on this task, so I guess that you checked with @sbuhl in private? What was the conclusion?

Reply:
@oussamamessaoudi Forgot to answer. 
Keep the current wording 👼